setup.py

```diff
@@ -5,11 +5,26 @@
 
 ext_modules = [
     Extension('ffm', ['fastFM/ffm.pyx'],
-              libraries=['m', 'fastfm'],
-              library_dirs=['fastFM/', 'fastFM-core/bin/'],
+              libraries=['m', 'fastfm', 'openblas'],
+              library_dirs=['fastFM/', 'fastFM-core/bin/',
+                            'fastFM-core/externals/OpenBLAS/',],
               include_dirs=['fastFM/', 'fastFM-core/include/',
                             'fastFM-core/externals/CXSparse/Include/',
-              numpy.get_include()])]
+                            'fastFM-core/externals/OpenBLAS/',
+                            numpy.get_include()]),
+    Extension('ffm2', ['fastFM/ffm2.pyx'],
+              libraries=['fastFMd', 'pthread', 'glog'],
+              library_dirs=['fastFM2/fpic/fastFM',
+                            '/home/ben/.hunter/_Base/033a6ff/1a47c45/e1266bb/Install/lib/'],              
```

Comment:
we can replace `/home/ben/.hunter/_Base/033a6ff/1a47c45/e1266bb/Install/lib/` with `hunter_base + '/Install/lib/'`.
I guess we will point `hunter_base` to a download dir with the pre-compiled binaries.